scripts/confirm_release.sh

```diff
@@ -25,7 +25,7 @@ echo "Download checksum file ..."
 curl --fail --output "$CHECKSUM_FILE" "${MEDIA_URL_PREFIX}/pgp/${CHECKSUM_FILE}"
 
 echo "Verify checksum file ..."
-if [ -n "${GPG_KEY}" ] ; then
+if [ -n "${GPG_KEY:-}" ] ; then
```

Comment:
I'm not familiar with this syntax, can you throw me a hint?

Reply:
Sure! This bash syntax is used to set a default if the `GPG_KEY` var is not defined. Without it, I was getting:
```
scripts/confirm_release.sh: line 28: GPG_KEY: unbound variable
```
since we are running the scripts in a pedantic-ish manner where `set -u` complains about undefined variables.